cherrypy/_cpreqbody.py

```diff
@@ -802,7 +843,13 @@ def read(self, size=None, fp_out=None):
             # Check lengths.
             self.bytes_read += datalen
             if self.maxbytes and self.bytes_read > self.maxbytes:
-                raise cherrypy.HTTPError(413)
+                if self.maxbytes_extend:
+                    print("extending maxbytes")
+                    self.maxbytes = self.maxbytes_extend()
+                    #We only want to call maxbytes extend once per request I think
+                    self.maxbytes_extend =None
+                if self.maxbytes and self.bytes_read > self.maxbytes:
```

Comment:
Do `bytes_read` and `maxbytes` mutate since previous check? If not, this conditional doesn't make any sense, because it's always `True` at this point.